tools/Microsoft.Health.Tools.Sql.Tasks/Helpers/SqlScriptWriter.cs

```diff
@@ -22,7 +22,7 @@ public SqlScriptWriter(string path)
         Directory.CreateDirectory(Path.GetDirectoryName(path));
 
         _writer = File.CreateText(path);
-        _sqlScriptGenerator = new Sql140ScriptGenerator();
+        _sqlScriptGenerator = new Sql160ScriptGenerator();
```

Comment:
I take it this refers to SQL Server 16 which is 2022 and I think is in preview. Could we use the current production version 15 instead? Is there a `Sql150ScriptGenerator`?

Reply:
Ok, I will try and verify Sql150ScriptGenerator.